src/main/java/by/custom/utilcalculator/controller/Bot.java

```diff
@@ -0,0 +1,139 @@
+package by.custom.utilcalculator.controller;
+
+import by.custom.utilcalculator.repository.resources.Commands;
+import by.custom.utilcalculator.repository.steps.*;
+import by.custom.utilcalculator.service.MessagesChecker;
+import by.custom.utilcalculator.service.MessagesCreator;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+public class Bot extends TelegramLongPollingBot {
+    private CountryOrigin countryOrigin = null;
```

Comment:
Please, test if two users simultaneously use the Bot. I think the conflict is possible.

Reply:
I didn't test it. But i am sure this problem will be, because we have only one singleton with fields. I will fix it soon (i hope)